### PR TITLE
[DONE] 14.0 improve compute folio sale lines and name invoice lines on change qty

### PR DESCRIPTION
**Pull request by miguelpadin**

### CODE REVIEW

**Comment by miguelpadin** [comment]
https://user-images.githubusercontent.com/6873385/114888607-b09e7100-9e09-11eb-802d-e200c1fe133f.mp4

File: pms/models/account_move_line.py [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
     # Fields declaration
+    # TODO: REVIEW why not a Many2one?
```
Comment (miguelpadin) [inline pms/models/account_move_line.py:11]: @DarioLodeiros

File: pms/models/account_move_line.py [hunk 2]
```diff
@@ -23,6 +24,50 @@ class AccountMoveLine(models.Model):
...
+        for record in self:
+            record.name = self.env["folio.sale.line"].generate_folio_sale_name(
+                record.folio_line_ids.reservation_id,
+                record.product_id,
+                record.folio_line_ids.service_id,
+                record.folio_line_ids.reservation_line_ids,
+                record.folio_line_ids.service_line_ids,
+                qty=record.quantity,
+            )
+            # TODO: check why this code doesn't work
```
Comment (miguelpadin) [inline pms/models/account_move_line.py:61]: @DarioLodeiros

File: pms/models/account_move_line.py (hunk 1 above)
Comment (miguelpadin) [inline pms/models/account_move_line.py:11]: @DarioLodeiros

**Review by DarioLodeiros — APPROVED** [review]

**Pull request merged**